src/common/src/config.rs

```diff
@@ -478,6 +478,14 @@ pub struct StreamingConfig {
 
     #[serde(default, flatten)]
     pub unrecognized: Unrecognized<Self>,
+
+    #[serde(default = "default::streaming::compute_runtime_worker_threads")]
+    /// compute runtime worker threads
+    pub compute_runtime_worker_threads: usize,
```

Comment:
This should be moved to `BatchConfig`, since this is used for configuring executions in batch.